{"uuid":"f483a120-e02a-4dba-93bd-22487ba269fd","name":"Scenario Outline: The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","children":["584494d1-4242-49ed-84e5-4ff2812129da"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382248605,"stop":1673382248607}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382249009,"stop":1673382249009},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382249673,"stop":1673382249673},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382249893,"stop":1673382249893},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673382249908,"stop":1673382249908}],"start":1673382248605,"stop":1673382249908}